spec/visual/templates/lexer_juxtaposed.erb

```diff
@@ -0,0 +1,52 @@
+<style type="text/css">
```

Comment:
I was able to rationalise the CSS a little further like so:

```css
  main {
    display: inline-block;
    margin-right: -215px;
    padding: 8px 215px 0 0;
    width: 100%;
  }
  .codeviewer {
    display: inline-block;
    padding: 8px 12px;
    width: 49.5%;
  } 
  .legend {
    float: right;
    padding: 16px 20px 8px 0;
    width: 215px; 
  } 
  .legend .codehilite { font-size: 13px }
```
This looked good in Chrome and Safari and avoided the problem (at least in Safari) where there would be a small amount of horizontal scroll for `div.codeviewer` containers.

How does it look on your end?

Reply:
Strange. I wonder why I didn't see the misalignment in the headers earlier today.
I had kept the horizontal scroll as a fall back measure. But looks like that won't be needed either.